### PR TITLE
BAU: Work Around stepfunction local testing

### PR DESCRIPTION
**Pull request by jkunle**
## Proposed changes

This is a possible work Around the step function local in ability to deal with map

### CODE REVIEW

File: step-functions/nino_issue_credential.asl.json [hunk 1]
```diff
@@ -153,7 +130,16 @@
         "credentialSubject": {
           "name": [
             {
-              "nameParts.$": "$.map"
+              "nameparts": [
+                {
+                  "type.$": "States.ArrayGetItem($.userDetails.Items[0].names.L[*].M.nameParts.L[?(@.M.type.S==GivenName)][*].type.S, 0)",
```
Comment (surajkumar) [inline step-functions/nino_issue_credential.asl.json:135]: What does the output of this look like?

Comment (jkunle) [reply step-functions/nino_issue_credential.asl.json:135]: Check your state machine -> `suraj-pdv-matching-NinoIssueCredential`

Comment (surajkumar) [reply step-functions/nino_issue_credential.asl.json:135]: The previous execution isn't there anymore, I'll redeploy these changes and have a test

File: step-functions/nino_issue_credential.asl.json [hunk 2]
```diff
@@ -153,7 +130,16 @@
...
+                  "value.$": "States.ArrayGetItem($.userDetails.Items[0].names.L[*].M.nameParts.L[?(@.M.type.S==GivenName)][*].value.S, 0)"
+                },
+                {
+                  "type.$": "States.ArrayGetItem($.userDetails.Items[0].names.L[*].M.nameParts.L[?(@.M.type.S==FamilyName)][*].type.S, 0)",
+                  "name.$": "States.ArrayGetItem($.userDetails.Items[0].names.L[*].M.nameParts.L[?(@.M.type.S==FamilyName)][*].value.S,0)"
```
Comment (surajkumar) [inline step-functions/nino_issue_credential.asl.json:140]: this needs to be `value.$` not `name.$`

**Review by surajkumar — CHANGES_REQUESTED** [review]

File: step-functions/nino_issue_credential.asl.json (hunk 2 above)
Comment (surajkumar) [reply step-functions/nino_issue_credential.asl.json:140]: Before:
```
"nameParts": [
            {
              "type": "GivenName",
              "value": "Jim"
            },
            {
              "type": "FamilyName",
              "value": "Ferguson"
            }
          ]
```

After:
```
"nameparts": [
            {
              "type": "GivenName",
              "value": "Jim"
            },
            {
              "name": "Ferguson",
              "type": "FamilyName"
            }
          ]
```

(difference between before and after)

**Review by surajkumar — APPROVED** [review]

**Pull request merged**